site/about/hero.php

```diff
@@ -0,0 +1,14 @@
+<div class="banner">
+      <div class="placeholder-tall-image">
+        <img src="../assets/images/placeholder-image-rectangle-tall.png" alt="placeholder-image-rectangle">
+      </div>
+      <div class="placeholder-image">
+        <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
+      </div>
+      <div class="about-title-mobile">
```

Comment:
Hey @Merciless0, unrelated to the php stuff, but I want you to try and not make duplicates of things for mobile and desktop. The better way to go about this would be to just have:

```
<div class="about-title">
    <h1><?php print $text; ?></h1>
</div>
```

in your HTML. Then in your css you can style that class in mobile and desktop. Lets chat about it during studio!